### PR TITLE
TedLium-V2 Dataset

### PR DESCRIPTION
**Pull request by christophmluscher**
PR for only the data of TedLium-V2.
Split from #111

### CODE REVIEW

File: common/datasets/tedlium2/lexicon.py [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+        compressed=True,
+    )
+    convert_lexicon_job.add_alias(os.path.join(output_prefix, "convert_text_to_bliss_lexicon_job"))
+
+    return convert_lexicon_job.out_bliss_lexicon
+
+
+@lru_cache()
+def get_bliss_lexicon(
+    output_prefix="datasets",
```
Comment (michelwi) [inline common/datasets/tedlium2/lexicon.py:73]:
```suggestion
    output_prefix: str = "datasets",
```

File: common/datasets/tedlium2/lexicon.py [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+        compressed=True,
+    )
+    merge_lexicon_job.add_alias(os.path.join(output_prefix, "merge_lexicon_job"))
+
+    return merge_lexicon_job.out_bliss_lexicon
+
+
+@lru_cache()
+def get_g2p_augmented_bliss_lexicon(
+    output_prefix="datasets",
```
Comment (michelwi) [inline common/datasets/tedlium2/lexicon.py:97]:
```suggestion
    output_prefix: str = "datasets",
```

**Review by Atticus1806 — COMMENTED** [review]
PR looks good to me, one thing to discuss, otherwise will be approved.

**Review by JackTemaki — CHANGES_REQUESTED** [review]
Docstrings for every function would be nice, but the important thing is the dataclass->dict conversion which should be removed.

File: common/datasets/tedlium2/download.py [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+
+from sisyphus import tk
+
+from i6_core.datasets.tedlium2 import (
+    DownloadTEDLIUM2CorpusJob,
+    CreateTEDLIUM2BlissCorpusJob,
+)
+
+
+@dataclass()
```
[the remark this answers is not in the document]
Comment (christophmluscher) [reply common/datasets/tedlium2/download.py]: using the class directly now

**Review by DanEnergetics — APPROVED** [review]
All looks good to me.

**Comment by Atticus1806** [comment]
So @JackTemaki all changes adressed? From my side PR is approved, but since you asked for changes I will let you make the last call.

**Review by JackTemaki — APPROVED** [review]
I have to trust you on the lexicon/cart, but in formality all seems fine now.

**Comment by JackTemaki** [comment]
@christophmluscher @Atticus1806 do you create the folder under `/work/common` or do you want to first see how the pipeline is running until we push the dataset "officially"?

**Comment by Atticus1806** [comment]
For now I would run it locally because I think it is not fully functional yet. Datasets should be fine, but recognition results do not seem correct yet.

**Comment by JackTemaki** [comment]
Okay then lets wait and check if we need a revision also of the data.

**Comment by christophmluscher** [comment]
@Atticus1806 can we merge this for now? the pipeline is running and we can still do further PRs to change things

**Comment by Atticus1806** [comment]
From my end yes.

**Pull request merged**